.travis.yml

```diff
@@ -28,3 +27,4 @@ install:
 
 script:
   - vendor/bin/phpspec run
+  - vendor/bin/phpunit tests
```

Comment:
Apparently  even if the testsuites are configured in the xml it still needs me to specify the directory (ex.: `.`).